Dockerfile

```diff
@@ -1,24 +1,10 @@
-FROM node:6.10
-RUN apt-get update -qq && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+FROM bitnami/node:8
 
-# Set up working directory
-RUN mkdir /app
+RUN npm install -g yarn@1.5.1
 
-# Set up node modules
-WORKDIR /tmp
-ADD package.json package.json
-ADD yarn.lock yarn.lock
-RUN npm i yarn -g
-RUN yarn install
-RUN cp -a /tmp/node_modules /app/
-
-# Finally, add the rest of our app's code
-# (this is done at the end so that changes to our app's code
-# don't bust Docker's cache)
 ADD . /app
-WORKDIR /app
 
-ENV PORT 5000
-EXPOSE 5000
+WORKDIR /app
+RUN yarn install
 
-CMD yarn start
+CMD yarn start
```

Comment:
I would work around this script called by `yarn start` https://github.com/saolsen/force/blob/acd564cce0a694eb5f406504cf8c563c1d835f9e/scripts/start.sh  and not load node with the `dotenv/config` module as well as not execute it with `forever`

Reply:
`node -r dotenv/config --max_old_space_size=1024 ./src`: note the `--max_old_space` prop; this needs to be maintained.

Reply:
This specifically helps during development when webpack is compiling assets (but not sure if we'll be using this for dev due to yarn link issues)

Reply:
In that case it can be overridden when running dev mode via `hokusai dev start` in `./hokusai/development.yml` - `command` key